.ci/static-checks.sh

```diff
@@ -299,6 +299,27 @@ static_check_go_arch_specific()
 	eval "$linter" "${linter_args}" "$dirs"
 }
 
+static_check_rust_arch_specific()
```

Comment:
I dont see where is called this function

Reply:
in the line type -F xxxx

Reply:
should this function be added to https://github.com/kata-containers/tests/blob/master/.ci/static-checks.sh#L1020-L1040

Reply:
in addition, this may fail if it runs on other distro than ubuntu, so I would recommend to add a validation to skip this if on other distro.

Reply:
no, it won't

Reply:
We won't set RUST_AGENT to yes on other distro, if it is set, it's a bug, this should fail

Reply:
ok, agree